src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImpl.java

```diff
@@ -282,17 +282,19 @@ protected void createAndRouteAccountingDocumentFromXml(AccountingXmlDocumentEntr
     }
     
     private void logClassLoaderDebugInfo() {
+        LOG.info("logClassLoaderDebugInfo:: OJB Broker ClassHelper.ClassLoader.name: {}", org.apache.ojb.broker.util.ClassHelper.getClassLoader());

```

Comment:
The log message labels are inconsistent between the various modified methods. You may want to update them for consistency, especially since your latest changes are logging the ClassLoader and not its "name()" call.